accounts/views.py

```diff
@@ -144,6 +144,13 @@ def complete_profile(request):
     # if user.desig == 'v':
     if request.method == 'POST':
         roll_no = request.POST['roll_no']
+        # To check whether entered roll number is already existed or not?
+        context = {}
```

Comment:
You don't need to initiate an empty dict here. You can simply create a new dict on line 151 if duplicate roll num is encountered.

Reply:
I think this won't be needed anymore as we are planning to not refresh the web page at any cost.

Reply:
Hmm, if we put up a synchronous call to check duplicate roll number every time the submit button is clicked, then we won't need this.